pkgs/development/tools/pnpm/fetch-deps/default.nix

```diff
@@ -82,6 +82,7 @@
                 --ignore-scripts \
                 ${lib.escapeShellArgs filterFlags} \
                 ${lib.escapeShellArgs pnpmInstallFlags} \
+                --registry="$NIX_NPM_REGISTRY" \
```

Comment:
I assume this is a no-op when this variable is unset/empty?

Reply:
I tested it and it just uses the default registry.